solving bug in jaxlib's tensorflow dependency by providing it with correct env variables
Binary files jax_orig/.git/index and jax/.git/index differ
diff -ruN jax_orig/WORKSPACE jax/WORKSPACE
--- jax_orig/WORKSPACE	2021-07-30 10:18:43.503078000 +0200
+++ jax/WORKSPACE	2021-08-05 11:44:13.187308000 +0200
@@ -7,18 +7,14 @@
 #    and update the sha256 with the result.
 http_archive(
     name = "org_tensorflow",
-    sha256 = "de22145c9dc51b3ed9399dbee2ab681f094f09e5213d6819451da500f163a14b",
+    # sha256 = "de22145c9dc51b3ed9399dbee2ab681f094f09e5213d6819451da500f163a14b",
     strip_prefix = "tensorflow-8cc3ffa8d8e4dd659c1534849cf5984ef4ec3532",
     urls = [
         "https://github.com/tensorflow/tensorflow/archive/8cc3ffa8d8e4dd659c1534849cf5984ef4ec3532.tar.gz",
     ],
+    patch_cmds=['sed -i -e "s|arguments = \[args\],|arguments = \[args\],\\n\\tuse_default_shell_env = True,|" third_party/mlir/tblgen.bzl']
 )
 
-# For development, one can use a local TF repository instead.
-# local_repository(
-#    name = "org_tensorflow",
-#    path = "tensorflow",
-# )
 
 load("//third_party/pocketfft:workspace.bzl", pocketfft = "repo")
 pocketfft()
